Expensify/App
# Show Closed Rooms as "Deleted" in E.Cash

@TomatoToaster pleas review

HOLD ON https://github.com/Expensify/Auth/pull/5781

### Details
Show Closed (status=2, state=2) Default Chat rooms as "Deleted" in E.Cash

### Fixed Issues
For https://github.com/Expensify/Expensify/issues/165433

### Tests/QA
- Create a new Policy on the legacy (oldDot) app, invite some users to it (make sure it's on an expensify domain so you have access to the beta)
- Open up E.cash, make sure that you can load up the #announce and #admins chat reports for that policy
- Delete the policy
- Reload the E.cash app, make sure that the #announce/admins rooms now show up as deleted:
<img width="1341" alt="Screen Shot 2021-07-13 at 4 26 52 PM" src="https://user-images.githubusercontent.com/4741899/125537886-c8a19775-a6d9-4518-a852-9ab169289496.png">
   - Make sure that the icon for the room has been replaced with a trash can icon
   - Make sure that the room shows up with `(deleted)` in the title
   - Make sure that the room input is disabled
   - Make sure that the room detail section does not contain any editable actions about the room (should just show an empty page)

### Tested On


### Screenshots


#### Web

<img width="374" alt="Screen Shot 2021-07-13 at 4 31 12 PM" src="https://user-images.githubusercontent.com/4741899/125538123-ca6cfa54-a003-4d1e-97eb-8b0ea1b4658d.png">
<img width="367" alt="Screen Shot 2021-07-13 at 4 30 53 PM" src="https://user-images.githubusercontent.com/4741899/125538124-154f123f-02ba-4228-985e-e2a07f8c8802.png">
<img width="246" alt="Screen Shot 2021-07-13 at 4 30 55 PM" src="https://user-images.githubusercontent.com/4741899/125538125-94b60427-3a29-43dd-aa45-982c0f757528.png">
<img width="262" alt="Screen Shot 2021-07-13 at 4 31 03 PM" src="https://user-images.githubusercontent.com/4741899/125538126-fbe67c20-2070-4ea7-a165-f23695b07151.png">
<img width="968" alt="Screen Shot 2021-07-13 at 4 31 07 PM" src="https://user-images.githubusercontent.com/4741899/125538127-be626d69-de28-42e5-ba63-042ee94caf8b.png">

#### Mobile Web


#### Desktop


#### iOS


#### Android
